README.rst

```diff
@@ -119,6 +119,28 @@ Event handlers are any callable where the first argument is an instance of the :
                               min_iterations,
                               lookback=5)
 
+Note on EXCEPTION_RAISED
+^^^^^^^^^^^^^^^^^^^^^^^^
+By default :code:`Ignite` will re-raise any exception caught during training. If you would like to change this
+behaviour, you have to register an handler for the :code:`EXCEPTION_RAISED` event. This handler will be called with
+the :code:`Engine` object and the caught :code:`Exception` as two first arguments, followed by any :code:`*args` and
+:code:`**kwargs` passed to :code:`add_event_handler` call. If this handler is registered, :code:`Ignite` will not
+re-raise the exception.
+
+The following example shows how to register a handler which will save the model before re-rasing the exception.
+
+.. code-block:: python
+
+    from ignite.engine import Events
+
+    def save_and_raise(engine, exception, model, path):
+        torch.save(model, path)
+        raise exception
+
+    trainer.add_event_handler(Events.EXCEPTION_RAISED,
+                              my_model,
```

Comment:
you forgot to add `save_and_raise` here

Reply:
Thanks, the example doesn't make too much sense without this lol.